frontend/src/layout/header/Logo.tsx

```diff
@@ -34,45 +35,50 @@ export const Logo: React.FC = () => {
     const alt = t("general.logo-alt", { title: translatedConfig(CONFIG.siteTitle, i18n) });
 
     return (
-        <Link
-            to="/"
+        <WithTooltip
+            tooltip={t("general.goto-homepage")}
+            placement="bottom"
```

Comment:
I would *probably* move this a little closer to the logo. But I won't protest if you think the distance is fine.

Reply:
Yeah I do like it better with the added distance. If it were completely up to me I would increase it even more, so that the tooltip is partly inside the box. This might not work for all logos though (or would it, with the logo always having a certain padding?).
<img width="236" alt="Bildschirm­foto 2023-03-09 um 17 25 49" src="https://user-images.githubusercontent.com/94838646/224088263-7027b822-b99d-4405-8f8e-1c25eda79ba4.png">
Hm on second thought this also looks a bit awkward and I imagine it would be hard to always find the right spot/distance for the tooltip, so keeping it outside the box is probably for the best.

Reply:
Yeah it's pretty hard to find a good distance. The two lines there are just.. meh. I pushed a version where the tooltip is on the right. This is an experiment, but if you like it we can merge. Idea being: right of the logo there should always be enough space (due to menu buttons) such that it can be shown. I think.